xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-api/src/test/java/org/xwiki/realtime/internal/DefaultRealtimeEditorManagerTest.java

```diff
@@ -0,0 +1,256 @@
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.xwiki.realtime.internal;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+
+import javax.inject.Provider;
+import javax.script.ScriptContext;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.xwiki.container.Container;
+import org.xwiki.container.Request;
+import org.xwiki.model.reference.DocumentReference;
+import org.xwiki.netflux.EntityChannel;
+import org.xwiki.netflux.EntityChannelStore;
+import org.xwiki.rendering.syntax.Syntax;
+import org.xwiki.script.ScriptContextManager;
+import org.xwiki.script.service.ScriptService;
+import org.xwiki.sheet.SheetManager;
+import org.xwiki.test.annotation.BeforeComponent;
+import org.xwiki.test.junit5.mockito.ComponentTest;
+import org.xwiki.test.junit5.mockito.InjectComponentManager;
+import org.xwiki.test.junit5.mockito.InjectMockComponents;
+import org.xwiki.test.junit5.mockito.MockComponent;
+import org.xwiki.test.mockito.MockitoComponentManager;
+import org.xwiki.wysiwyg.script.WysiwygEditorScriptService;
+
+import com.xpn.xwiki.XWiki;
+import com.xpn.xwiki.XWikiContext;
+import com.xpn.xwiki.doc.XWikiDocument;
+
+@ComponentTest
+class DefaultRealtimeEditorManagerTest
+{
+
+    private static final String WIKI = "wiki";
+
+    private static final String WYSIWYG = "wysiwyg";
+
+    private static final String EDITOR = "editor";
+
+    private static final String EDIT = "edit";
+
+    private static final String EN = "en";
+
+    private static final String CONTENT = "content";
+
+    @InjectMockComponents
+    private DefaultRealtimeEditorManager realtimeEditorManager;
+
+    @MockComponent
+    private Provider<XWikiContext> xwikiContextProvider;
+
+    @MockComponent
+    private EntityChannelStore entityChannelStore;
+
+    @MockComponent
+    private SheetManager sheetManager;
+
+    @MockComponent
+    private ScriptContextManager scriptContextManager;
+
+    @MockComponent
+    private Container container;
+
+    @MockComponent
+    private Request request;
+
+    @Mock
+    private ScriptContext scriptContext;
+
+    @Mock
+    private XWikiContext xwikiContext;
+
+    @Mock
+    private XWikiDocument document;
+
+    @Mock
+    private XWiki wiki;
+
+    @Mock
+    EntityChannel channel;
+
+    @InjectComponentManager
+    MockitoComponentManager componentManager;
+
+    private WysiwygEditorScriptService wysiwygEditorScriptService;
+
+    private List<String> path;
+
+    @BeforeComponent
+    void beforeComponent() throws Exception
```

Comment:
```suggestion
    void beforeComponent(MockitoComponentManager componentManager) throws Exception
```

You can inject it as method parameter, if you don't use it elsewhere.